graylog2-web-interface/src/components/common/TimezoneSelect.jsx

```diff
@@ -12,25 +12,33 @@ const TimezoneSelect = React.createClass({
   getValue() {
     return this.refs.timezone.getValue();
   },
+
+  // Some time zones are not stored into any areas, this is the group we use to put them apart in the dropdown
+  // https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+  _UNCLASSIFIED_AREA: 'Unclassified',
+
   _formatTimezones() {
     const timezones = {};
     moment.tz.names().forEach((timezone) => {
```

Comment:
Just a remark: Wouldn't it be better to export [the list of supported time zones in Joda-Time](http://joda-time.sourceforge.net/timezones.html) once and use that instead of what [Moment.js](http://momentjs.com/) returns?